### ISSUE TITLE
Parse error when requiring json file

### ISSUE THREAD

**Issue by khaney** [zsoltszabo/node-uglifier#3]
Just thought I'd leave a note.  I assume this is because of a json file.  If not, please disregard.
````
lib\app.js added to sources
lib\server.js added to sources
config.json added to sources

Error
    at new JS_Parse_Error (C:\Dev\desktop-master-service\node_modules\node-uglifier\node_modules\uglify-js\lib\parse.js:189:18)
    at js_error (C:\Dev\desktop-master-service\node_modules\node-uglifier\node_modules\uglify-js\lib\parse.js:197:11)
    at croak (C:\Dev\desktop-master-service\node_modules\node-uglifier\node_modules\uglify-js\lib\parse.js:657:9)
    at token_error (C:\Dev\desktop-master-service\node_modules\node-uglifier\node_modules\uglify-js\lib\parse.js:665:9)
    at unexpected (C:\Dev\desktop-master-service\node_modules\node-uglifier\node_modules\uglify-js\lib\parse.js:671:9)
    at semicolon (C:\Dev\desktop-master-service\node_modules\node-uglifier\node_modules\uglify-js\lib\parse.js:691:43)
    at simple_statement (C:\Dev\desktop-master-service\node_modules\node-uglifier\node_modules\uglify-js\lib\parse.js:865:73)
    at C:\Dev\desktop-master-service\node_modules\node-uglifier\node_modules\uglify-js\lib\parse.js:724:47
    at C:\Dev\desktop-master-service\node_modules\node-uglifier\node_modules\uglify-js\lib\parse.js:704:24
    at block_ (C:\Dev\desktop-master-service\node_modules\node-uglifier\node_modules\uglify-js\lib\parse.js:978:20)
````

**Comment by zsoltszabo** [comment]
Hi for the AST tree I use UglifyJs parse. It seems it does not support JSON format since it is not valid Javascript. See https://github.com/SummerWish/express-minify/issues/1
I could hack something into my code to enclose the json part with parenthesis in the source input of my
ast=packageUtils.getAst(source) which just calls UglifyJs.parse...

**Comment by zsoltszabo** [comment]
Ok I pushed a patch to support json. The source is now modified to look like a regular js module.

**Comment by khaney** [comment]
Cool.  Nice work.

On Wed, Mar 4, 2015 at 7:15 PM, zsoltszabo <notifications@github.com> wrote:

> Closed #3 <https://github.com/zsoltszabo/node-uglifier/issues/3>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zsoltszabo/node-uglifier/issues/3#event-246464141>.
>



-- 
*Kris Haney*
Software Engineer - Client Architecture
Perceptive Software | from Lexmark

**Comment by zsoltszabo** [comment]
You are welcome!
